cmd/snap-update-ns/change.go

```diff
@@ -66,9 +65,11 @@ var changePerform = (*Change).Perform
 // this change (such as mounted tmpfs or overlayfs).
 func (c *Change) Perform() ([]*Change, error) {
 	if c.Action == Mount {
-		mode := os.FileMode(0755)
-		uid := 0
-		gid := 0
+		const (
```

Comment:
Why are we doing this? Those are likely to be variables (via the x-snapd.{mode,uid,gid} options).

Reply:
This was explained on IRC, this essentially allows us to skip importing the right package to get the right types as constants have this done automatically by golang.